docs/core_docs/docs/integrations/document_loaders/web_loaders/firecrawl.mdx

```diff
@@ -0,0 +1,45 @@
+---
+hide_table_of_contents: true
+---
+
+# Firecrawl
+
+This guide shows how to use [Firecrawl](https://firecrawl.dev) with LangChain to load web data into an LLM-ready format using Firecrawl.
+
+## Overview
+
+[FireCrawl](https://firecrawl.dev) crawls and convert any website into LLM-ready data. It crawls all accessible subpages and give you clean markdown and metadata for each. No sitemap required.
+
+FireCrawl handles complex tasks such as reverse proxies, caching, rate limits, and content blocked by JavaScript. Built by the [mendable.ai](https://mendable.ai) team.
+
+This guide shows how to scrap and crawl entire websites and load them using the `FireCrawlLoader` in LangChain.
+
+## Setup
+
+You'll need to sign up and retrieve your [FireCrawl API key](https://firecrawl.dev).
```

Comment:
```suggestion
You'll need to sign up and retrieve your [FireCrawl API key](https://firecrawl.dev).
```
I would add some mention of the 100 free credits. I could easily see a user reading up until the API key requirement part and getting discouraged before they're even able to try it out

Reply:
100%